bindings/binding_core_wasm/src/types.rs

```diff
@@ -793,14 +793,16 @@ export interface EsParserConfig {
   importAssertions?: boolean;
 }
 
+type JSXPreset = "react" | "react-jsx" | "react-jsxdev" | "preserve" | "react-native";
+
 /**
  * Options for transform.
  */
 export interface TransformConfig {
   /**
-   * Effective only if `syntax` supports ƒ.
+   * Effective only if `syntax` supports.
    */
-  react?: ReactConfig;
+  react?: JSXPreset | ReactConfig;
```

Comment:
The `react` field now supports both object-style configuration (maintaining Babel compatibility) and string-based presets (like TypeScript's JSX presets). 
However, the current config `"react": "react"` appears slightly odd. We hope to rename this field to `jsx` during future configuration refinements.

Reply:
👍